psi4/src/psi4/libfock/jk.h

```diff
@@ -1191,7 +1171,143 @@ class PSI_API MemDFJK : public JK {
     std::shared_ptr<DFHelper> dfh() { return dfh_; }
 };
 
+class JBase {
```

Comment:
`JBase` and `KBase` do the exact same thing and therefore code duplication. What about keeping just one of them and renaming it to `JorKBase` or something similar?

Reply:
Made the change. Two classes are now merged in `SplitJKBase`